### ISSUE TITLE
PMDEmptyTest.java:48-50: Tests below pass only when...

### ISSUE THREAD

**Issue by davvd** [teamed/qulice#596]
Puzzle `544-c37b2cc3` in `qulice-pmd/src/test/java/com/qulice/pmd/PMDEmptyTest.java:48-50` has to be resolved: Tests below pass only when run sequentially, when they are run in parallel some of them start to fail. Please fix the tests below and remove override of maven-surefire-plugin configuration in qulice-pmd pom.xml... The puzzle was created by Prahlad Yeri on 6-Jan-2016.

If you have any technical questions, don't ask me, submit new tickets instead

**Comment by krzyk** [comment]
@davvd this is postponed

**Comment by davvd** [comment]
> @davvd this is postponed 

@krzyk right, I added "postponed" label

**Comment by davvd** [comment]
> @davvd this is postponed 

@krzyk I will assign somebody else to this issue

**Comment by davvd** [comment]
@krzyk Closing this issue now...
